src/ui/src/core/index.ts

```diff
@@ -59,6 +59,12 @@ export function generateCore() {
 		addMailSubscription("pageChange", (pageKey: string) => {
 			setActivePageFromKey(pageKey);
 		});
+		addMailSubscription("localStorageSetItem", (event) => {
+			localStorage.setItem("wf." + event.key, JSON.stringify(event.value));
+		});
+		addMailSubscription("localStorageRemoveItem", (event) => {
+			localStorage.removeItem("wf." + event.key);
+		});
```

Comment:
You can add the type
```suggestion
		addMailSubscription("localStorageSetItem", (event: LocalStorageSetItemEvent) => {
			localStorage.setItem("wf." + event.key, JSON.stringify(event.value));
		});
		addMailSubscription("localStorageRemoveItem", (event: LocalStorageRemoveItemEvent) => {
			localStorage.removeItem("wf." + event.key);
		});
```